### ISSUE TITLE
table inside a button is strange

### ISSUE THREAD

**Issue by nareg26** [ArazAslanian/testFinal#6]
https://github.com/ArazAslanian/testFinal/blob/3f4120a673efbf27d9ef1f5d2fd81d63018a1e61/public/main.js#L174

**Comment by nareg26** [comment]
Can this already be in the html? And we fill the values with:
`$('.class-name').html(value)`

**Comment by nareg26** [comment]
Table inside a button is strange

**Comment by nareg26** [comment]
Seems some things can be in #mainList already in the html?

**Comment by ArazAslanian** [comment]
I don't think so because the amount of the tables / buttons created depends on how many data points we have in the database
